app/tests/src/common/tx-settings.js

```diff
@@ -91,6 +91,65 @@ const transactionSettings = {
     ttl,
     txHash: '059cce8e6c1d404798237b36ee45f74dafb0cf463ad236def4bf305474b6a666',
   },
+  sendAdaNullTtl: {
```

Comment:
worth noting that this test would pass even on current develop, the actual runtime changes were made just to the transformations of data in the crypto providers for hw wallets, but we are not testing them currently and I wouldn't introduce that just for this small modification. Still, it seems worthwhile explicitly covering that we can handle serialization/planning of a transaction with null TTL